overrides/android-override.json

```diff
@@ -645,5 +645,35 @@
         }
     },
     "unprotectedTemporary": [
-    ]
+    ],
+    "variantManager": {
```

Comment:
Let's make the naming here more clear. Something like `experimentVariantManager` will make it more clear what this section is for.

```suggestion
    "experimentVariantManager": {
```

Reply:
@SlayterDev I changed the name for `experimentalVariants` as we agreed on the Asana task 👍